centaurCwlRunner/src/main/scala/centaur/cwl/OutputManipulator.scala

```diff
@@ -61,20 +46,19 @@ object OutputManipulator extends Poly1 {
           schema <- tpe.select[OutputArraySchema]
           items = schema.items
           innerType <- items.select[MyriadOutputInnerType]
-          outputJson = metadata.map(resolveOutputViaInnerType(innerType)).asJson
+          outputJson = metadata.map(m => resolveOutputViaInnerType(innerType)(m, pathBuilder)).asJson
```

Comment:
ToL: If the pathBuilder parameter of `resolveOutputViaInnerType` is moved next to `innerType` instead you could keep it as  `metadata.map(resolveOutputViaInnerType(innerType, pathBuilder)).asJson`